activerecord/lib/active_record/core.rb

```diff
@@ -204,6 +219,8 @@ def self.global_executor_concurrency # :nodoc:
         @@global_executor_concurrency ||= nil
       end
 
+      mattr_accessor :application_record_class, instance_accessor: false, default: nil
```

Comment:
This looks like it got added accidentally in a rebase maybe?

Reply:
Yep looks that way, sorted now :)